app/assets/stylesheets/note.scss

```diff
@@ -0,0 +1,11 @@
+$grey: #808080;
```

Comment:
This is ok for now, but it would be nice to make a pass at consolidating our colors and sizes into a variables files eventually.